versions-enforcer/src/site/markdown/max-dependency-updates.md

```diff
@@ -0,0 +1,112 @@
+| `ruleSet`                                     |            | Allows specifying the `RuleSet` object describing rules on artifact versions to ignore when considering updates.<br/> See: [Using the ruleSet element in the POM](../version-rules.html#Using_the_ruleSet_element_in_the_POM)                                                    |
+
+**Note:** Inclusion/exclusion parameters like `dependencyIncludes`, `dependencyExcludes`, etc. work the same way as parameters
+of the same name of the [versions:display-dependency-updates](../display-dependency-updates-mojo.html) goal
+of the plugin.
+
+The parameters accept a list of *extended GAV* patterns, meaning patterns of:
+
+`groupId:artifactId:version:type:classifier:scope`
+
+of which only `groupId` is obligatory. On top of that, all of the components can be replaced with the asterisk (`*`)
+character in which case it will match all values.
+
+So, e.g. both of the below patterns:
+- `org.codehaus.mojo`
+- `org.codehaus.mojo:*`
+
+will match all artifacts with groupId `org.codehaus.mojo`.
+
+### Sample Plugin Configuration
+
+Below a rundimentary example of using the enforcer rule. The example assumes that the versions of both
+the enforcer plugin as well as the versions plugin are set elsewhere (using dependencyManagement).
+
+The below example specifies a rule which will not allow any updates except for updates of `localhost:dummy-api`.
+It will also ignore all sub-incremental updates.
+
+```xml
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    ...
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <goals>
+                    <goal>enforce</goal>
+                </goals>
+                <configuration>
+                    <rules>
+                        <maxDependencyUpdates>
+                            <maxUpdates>0</maxUpdates>
+                            <dependencyExcludes>
+                                <dependencyExclude>localhost:dummy-api</dependencyExclude>
+                            </dependencyExcludes>
+                            <ignoreSubIncrementalUpdates>true</ignoreSubIncrementalUpdates>
+                        </maxDependencyUpdates>
+                    </rules>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>versions-maven-plugin</artifactId>
```

Comment:
versions-enforcer

Reply:
Forgot to remove that -- I'm using integration tests instead.

Reply:
Didn't know what you meant at first. I first wanted to use the plugin itself for that -- that's why a `runtime` scope dependency on `versions-enforcer`.

But this will also do. So that it's possible to simply depend on `versions-enforcer` and not the whole plugin.